### PR TITLE
[ODS-6538-6x]  v2.3.5 docker patch release for  v6.2-patch2 ODS release

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Web-Ods-Api/Alpine/pgsql/Dockerfile [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.10-alpine3.20@sha256:09bd97b2f28a084168e6d4e2f1c32385753cd7166c8d87af71a0dba2919d7aa2 AS base
 RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8 && \
     addgroup -S edfi && adduser -S edfi -G edfi
 
 # Start a new layer so that the above layer can be cached
 FROM base AS build
-RUN apk --no-cache add postgresql13-client=~13
+RUN apk --no-cache add postgresql-client
```
Comment (semalaiappan) [inline Web-Ods-Api/Alpine/pgsql/Dockerfile:12]: postgresql13-client is not supported with aspnet:8.0.10-alpine3.20

File: Web-Sandbox-Admin/Alpine/pgsql/Dockerfile [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.10-alpine3.20@sha256:09bd97b2f28a084168e6d4e2f1c32385753cd7166c8d87af71a0dba2919d7aa2 AS base
 RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8 && \
     addgroup -S edfi && adduser -S edfi -G edfi
 
 # Start a new layer so that the above layer can be cached
 FROM base AS build
-RUN apk --no-cache add postgresql13-client=~13
+RUN apk --no-cache add postgresql-client
```
Comment (semalaiappan) [inline Web-Sandbox-Admin/Alpine/pgsql/Dockerfile:12]: postgresql13-client is not supported with aspnet:8.0.10-alpine3.20

File: Web-Ods-Api/Alpine/pgsql/Dockerfile [hunk 3]
```diff
@@ -3,17 +3,17 @@
...
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.10-alpine3.20@sha256:09bd97b2f28a084168e6d4e2f1c32385753cd7166c8d87af71a0dba2919d7aa2 AS base
 RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8 && \
     addgroup -S edfi && adduser -S edfi -G edfi
 
 # Start a new layer so that the above layer can be cached
 FROM base AS build
-RUN apk --no-cache add postgresql13-client=~13
+RUN apk --no-cache add postgresql16-client=~16
```
Comment (semalaiappan) [inline Web-Ods-Api/Alpine/pgsql/Dockerfile:12]: postgresql13-client is not supported with aspnet:8.0.10-alpine3.20 , that's why postgresql16-client

File: Web-Sandbox-Admin/Alpine/pgsql/Dockerfile [hunk 4]
```diff
@@ -3,17 +3,17 @@
...
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.10-alpine3.20@sha256:09bd97b2f28a084168e6d4e2f1c32385753cd7166c8d87af71a0dba2919d7aa2 AS base
 RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8 && \
     addgroup -S edfi && adduser -S edfi -G edfi
 
 # Start a new layer so that the above layer can be cached
 FROM base AS build
-RUN apk --no-cache add postgresql13-client=~13
+RUN apk --no-cache add postgresql16-client=~16
```
Comment (semalaiappan) [inline Web-Sandbox-Admin/Alpine/pgsql/Dockerfile:12]: postgresql13-client is not supported with aspnet:8.0.10-alpine3.20 , that's why postgresql16-client

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**